### PR TITLE
Merged download, verify, and decompress instructions

### LINKED ISSUE #631: Combine download, verify, and decompress into 1 doc
**Do you think Clear Linux documentation needs a new document? Please describe.**
Combine download, verify, and decompress into 1 doc, titled: `download-verify-decompress`

Please include the three different types of OSes in the new doc: 
- https://clearlinux.org/documentation/clear-linux/guides/maintenance/download-verify-decompress-linux
- https://clearlinux.org/documentation/clear-linux/guides/maintenance/download-verify-decompress-mac
- https://clearlinux.org/documentation/clear-linux/guides/maintenance/download-verify-decompress-windows

**Should the new document be a guide, a reference, or a tutorial?**
The new document will reside here, as before:
https://clearlinux.org/documentation/clear-linux/guides/maintenance

**Additional context**
Please complete by Thursday afternoon PDT. I'd like to reference the new consolidated document from `bare-metal-install-server` and `bare-metal-install-desktop`.

### PR DESCRIPTION
**Pull request by DougTW**
Linux, Mac, Windows instructions for download, verify, decompress CL image.

- closes #631

Signed-off-by: DougTW <doug.martin@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/maintenance/download-verify-decompress.rst [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+.. _download-verify-decompress-windows:
+
+
+Download, verify, and decompress a |CL-ATTR| image
+##################################################
+
+This guide describes the types of |CL| images available, where to download
+them, how to verify the integrity of an image, and how to decompress it.
+
+
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/download-verify-decompress.rst]: Would you add a local table of contents right after the intro sentence? Will make this standard w/ other guide updates. 

..contents:: 
  :local:
  :depth: 1

File: source/clear-linux/guides/maintenance/download-verify-decompress.rst [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+Image types
+***********
+
+.. include:: ../../reference/image-types.rst
+   :start-after: image-types-content:
+
+Instructions for each operating system
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/download-verify-decompress.rst]: Could we move the "follow instructions for your OS" bit to the intro and then do away with this heading entirely? Then update each OS set of instructions to use the same heading level as Image types (eg use ***** to underline). Then the page outline from the local TOC will read: 
-Image types
-Linux instructions
-macOS instructions
-Windows instructions

And users can jump to their section from the top if desired.

File: source/clear-linux/guides/maintenance/download-verify-decompress.rst [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+Verify the integrity of the |CL| image
+--------------------------------------
+
+Before you use a downloaded |CL| image, verify its integrity. This action
+eliminates the small chance of a corrupted image due to download issues. To
+support verification, each released |CL| image has a corresponding SHA512
+checksum file designated with the suffix `-SHA512SUMS`.
+
+#. Download the corresponding SHA512 checksum file of your |CL| image.
+#. Start the Terminal app.
```
Comment (mvincerx) [inline source/clear-linux/guides/maintenance/download-verify-decompress.rst]: Replace with: Open a terminal.

**Comment by mvincerx** [comment]
LGTM. Thanks for all the work on this, @DougTW !

**Pull request merged**